controller/kv/helper.go

```diff
@@ -3286,3 +3292,80 @@ func (m clusterHelper) GetSigstoreTimestamp() (string, *uint64, error) {
 
 	return string(configData), &rev, nil
 }
+
+func (m clusterHelper) CreateRemoteRepository(nickname string, remoteRepository *api.RemoteRepository, txn *cluster.ClusterTransact) error {
+	remoteRepositoryKey := share.CLUSRemoteRepositoryKey(nickname)
+
+	value, err := json.Marshal(remoteRepository)
+	if err != nil {
+		return err
+	}
+
+	if txn != nil {
+		txn.Put(remoteRepositoryKey, value)
+	} else {
+		cluster.Put(remoteRepositoryKey, value)
```

Comment:
should it be cluster.PutIfNotExist() called?

Reply:
Correct. Implemented.